src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -99,11 +99,11 @@ public ISymbolNode ReadyToRunHelperFromTypeLookup(ReadyToRunHelperId id, Object
             return _genericReadyToRunHelpersFromType.GetOrAdd(new ReadyToRunGenericHelperKey(id, target, dictionaryOwner));
         }
 
-        private NodeCache<ReadOnlyDataBlobKey, BlobNode> _readOnlyDataBlobs;
+        private NodeCache<ReadOnlyDataBlobKey, SettableBlobNode> _readOnlyDataBlobs;
```

Comment:
I didn't realize this was the only use of `BlobNode`. Can you also delete `BlobNode.cs`? It feels like something we will need again, but we can just restore it from history it's ever needed again.

Reply:
Sure. I deleted BlobNode.cs.